resources/views/accordion.blade.php

```diff
@@ -15,15 +15,19 @@
     'contentClass'    => 'mt-2',
 ])
 
-<div x-data="{ openPanel: null }">
+<div
+    class="accordion"
+    x-data="{ openPanel: false }"
+    :class="{ 'accordion-open': openPanel }"
+>
     <dl>
         <div class="{{ $containerClass }} @if ($dark === false && $border) border-2 border-theme-secondary-200 rounded-xl @endif">
             <dt>
                 <button
-                    class="text-left w-full flex justify-between items-center rounded
+                    class="accordion-trigger text-left w-full flex justify-between items-center rounded
```

Comment:
🤔 what `accordion-trigger` does exactly? because I didn't find the css class...

Reply:
It was used as an identifier: https://github.com/ArkEcosystem/nodem/pull/135/files#diff-6dd540c9e1bcdef978d2624d61e922900c97fe8f5cfcdc188729126202098164R37

but I've also moved the classes to the css to make use of it